src/main/java/seedu/address/model/widget/WidgetObject.java

```diff
@@ -40,140 +42,156 @@ public WidgetObject() {
     }
 
     /**
-     * Sets the content of the widget object.
+     * Sets the content of the widget object. Any fields not set will be an empty string.
      *
      * @param args All the fields to be set.
      */
-    public void set(String ...args) {
-        for (int i = 0; i < Math.min(args.length, 14); i++) {
+    public void set(String... args) {
+        // Sets all fields from start to end order.
+        for (int i = 0; i < Math.min(args.length, NUMBER_OF_FIELDS); i++) {
             divs[i] = args[i];
         }
     }
 
+    /**
+     * Sets the first null field to the given String value. Any fields not set will be an empty string.
+     *
+     * @param arg The String value to be set to the first null field.
+     */
+    public void set(String arg) {
+        System.out.println(arg);
```

Comment:
Is this sysprint leftover from debugging?

Reply:
Yea I using this method as the catch all during the GUI debugging. Actually good idea, I used logger logcenter instead.